dev/firely/docker-compose.mssqlserver.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+
+  firely:
+    depends_on:
+      - db
+    environment:
+      - VONK_Repository=SQL
+      - VONK_SqlDbOptions:ConnectionString=Initial Catalog=VonkData;Data Source=db,1433;User ID=sa;Password=SQLServerStrong(!)Password*
+      - VONK_SqlDbOptions:SchemaName=vonk
+      - VONK_SqlDbOptions:AutoUpdateDatabase=true
+      - VONK_SqlDbOptions:AutoUpdateConnectionString=Initial Catalog=VonkData;Data Source=db,1433;User ID=sa;Password=SQLServerStrong(!)Password*
+
+  db:
+    image: mcr.microsoft.com/mssql/server
+    ports:
+      - "1433:1433"
+    environment:
+      - ACCEPT_EULA=Y
+      - SA_PASSWORD=SQLServerStrong(!)Password*
+    healthcheck:
+      test: /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P 'SQLServerStrong(!)Password*' -Q 'SELECT 1 FROM VonkData.sys.tables'
```

Comment:
Very happy to see a healthcheck here! 

(self-reported) Healthchecks make monitoring MUCH easier.  Unfortunately it appears `/opt/mssql-tools/bin/sqlcmd` incorrectly returns a successful exit code (`0`) after a failure.

Re-running the healthcheck command directly inside the container shows `sqlcmd` incorrectly returning a successful exit code (`0`) when the SQL query fails:

```
$ dc exec db /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P 'SQLServerStrong(!)Password*' -Q 'SELECT 1 FROM VonkData.sys.tables'

Msg 208, Level 16, State 1, Server 5ad6dc0a78e3, Line 1
Invalid object name 'VonkData.sys.tables'.


echo $?
0
```


Not a huge issue, but would suggest removing until fixed since it'll return false positives

Reply:
Thanks for checking this out! I've removed it for now.